src/context/auth.rs

```diff
@@ -0,0 +1,39 @@
+use crate::{raw, Context, ValkeyString};
+use std::os::raw::{c_char, c_int};
+use std::ptr;
+
+impl Context {
+    /// Authenticates a client using an ACL user
+    ///
+    /// Wrapper around RedisModule_AuthenticateClientWithACLUser.
+    /// Available in Redis 7.2+ or Valkey 8.0+.
+    ///
+    /// # Arguments
+    /// * `username` - ACL username to authenticate with
+    ///
+    /// # Returns
+    /// * `Status::Ok` - Authentication successful
+    /// * `Status::Err` - Authentication failed
+    #[cfg(all(any(
+        feature = "min-redis-compatibility-version-7-0",
```

Comment:
Wha't about `min-redis-compatibility-version-7-2`?  Why not invert the logic to have 
```
    not(any(
        feature = "min-redis-compatibility-version-6-2",
        feature = "min-redis-compatibility-version-6-0"
    ))
```